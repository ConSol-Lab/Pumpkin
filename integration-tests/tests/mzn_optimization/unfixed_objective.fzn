var 1..3: objective :: output_var;
var 1..2: other :: output_var;

constraint int_lin_le([1, -1], [other, objective], -1);

solve :: int_search([other], smallest, indomain_min) maximize objective;