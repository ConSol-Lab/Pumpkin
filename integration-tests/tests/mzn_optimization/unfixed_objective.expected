objective = 3;
other = 1;
----------
==========